integration/src/main/java/scripts/utils/TestUtils.java

```diff
@@ -6,6 +6,7 @@ public class TestUtils {
   private static final Random RANDOM = new Random();
 
   public static String appendRandomNumber(String string) {
-    return string + "-" + RANDOM.nextInt(10000);
+    // Can't be dash because BQ dataset names can't have dash
```

Comment:
nit: would be nice in the future to cherry-pick these distinct fixes into new other PRs.

Reply:
This is just a test util, it's not changing application code